content/en/serverless/installation/nodejs.md

```diff
@@ -37,184 +37,139 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
+<div class="alert alert-warning">This service is in public beta. If you have any feedback, contact <a href="/help">Datadog support</a>.</div>
 
-    ```bash
-    aws cloudformation deploy \
-        --stack-name datadog-cfn-macro \
-        --template-file packaged.template \
-        --capabilities CAPABILITY_IAM
-    ```
+Use the Datadog CLI to set up instrumentation on your Lambda functions in your CI/CD pipelines. The CLI command automatically adds the Datadog Lambda library to your functions by using layers, and configures your functions to send metrics, traces, and logs to Datadog.
 
-The macro is now deployed and ready to use.
+### Install the Datadog CLI
 
-### Install the Datadog Lambda Library
+Install the Datadog CLI by using NPM or Yarn:
 
-Use the Datadog CloudFormation macro to ingest traces, logs, and enhanced metrics from your application without any code instrumentation. The macro automatically attaches the Datadog Lambda Library for Node.js and Python to your functions using layers. At deploy time, it generates new handler functions that wrap your existing functions and initializes the Lambda Library.
+```sh
+# NPM
+npm install -g @datadog/datadog-ci
 
-To install the Datadog Lambda Library with the CloudFormation macro, follow these steps:
+# Yarn
+yarn global add @datadog/datadog-ci
+```
 
-1. Add a `CfnMapping` to your `Stack` object in your AWS CDK app:
-  ```typescript
-      import * as cdk from "@aws-cdk/core";
+### Instrument the Function
 
-      class CdkStack extends cdk.Stack {
-        constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
-          super(scope, id, props);
-          this.addTransform("DatadogCfnMacro");
+Run the following command with your [AWS credentials][1]. Replace `<functionname>` and `<another_functionname>` with your Lambda function names, `<aws_region>` with the AWS region name, `<layer_version>` with the desired version of the Datadog Lambda layer (see [latest releases][2]) and `<forwarder_arn>` with Forwarder ARN (see the [Forwarder documentation][3]).
 
-          new cdk.CfnMapping(this, "Datadog", {
-            mapping: {
-              Parameters: {
-                forwarderArn: "arn:aws:lambda:us-east-1:000000000000:function:datadog-forwarder",
-                stackName: this.stackName,
-                enableDDTracing: true,
-                flushMetricsToLogs: true,
-              },
-            },
-          });
-        }
-      }
-  ```
-    Find your Datadog Forwarder ARN in the [AWS Console][2]. For more information on installing the Forwarder, see the [official documentation][3].
+```sh
+datadog-ci lambda instrument -f <functionname> -f <another_functionname> -r <aws_region> -v <layer_version> --forwarder	<forwarder_arn>
+```
 
-2. Redeploy your serverless application.
+For example:
 
+```sh
+datadog-ci lambda instrument -f my-function -f another-function -r us-east-1 -v 26 --forwarder arn:aws:lambda:us-east-1:000000000000:function:datadog-forwarder
```

Comment:
This looks right to me, but there is more documentation on parameters in the [Github readme of the commands](https://github.com/DataDog/datadog-ci/tree/master/src/commands/lambda).Might be worth linking to?

Reply:
Good point, I should add a link to the readme for other tools too.